.github/benchmark-configs.json

```diff
@@ -157,5 +166,6 @@
       "size": "Single-Node",
       "data_instance_config": "4vCPU, 32G Mem, 16G Heap"
     }
-  }
+  },
+  "baseline_cluster_config": "x64-r5.xlarge-single-node-1-shard-0-replica-baseline"
```

Comment:
This seems to be off, should be under `"id_10": { }` node

Reply:
sorry about that, fixed.